docs/variants/qemu_q35/building-manual.md

```diff
@@ -1,66 +1,88 @@
-# Dasharo (UEFI) v0.1.0 for QEMU Q35 - Building Manual
+# Building manual
 
-Follow below steps to create the "Dasharo (UEFI) OVMF image for QEMU Q35" from EDK2:
+## Intro
+
+This documents describes the procedure for compiling Dasharo firmware
+for QEMU Q35.
 
 ## Requirements
 
-- Docker
-    + follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
-    + follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+* `Ubuntu 20.04/21.04/22.04` as a host OS was tested
+* Internet connection
+* Docker installed
+    - follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+    - follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+* Git, wget, unzip installed
+
+```bash
+sudo apt install git unzip wget
+```
 
 ## Procedure
 
-Easiest way to build OVMF for QEMU Q35 is to use coreboot Docker image.
-As some dependencies maybe missing in given local instance.
+Obtain Dasharo source code:
 
-Obtain the docker image:
+> Replace the `REVISION` with one of the:
+>
+> * `qemu_q35_vVERSION` (e.g. `qemu_q35_v0.1.0`) for the given release
+> * `qemu_q35_vVERSION-rcN` (e.g. `qemu_q35_v0.1.0-rc1`) for the given release
+>   candidate
+
+Clone Dasharo EDK II fork:
 
 ```bash
-docker pull coreboot/coreboot-sdk:2021-09-23_b0d87f753c
+git clone https://github.com/Dasharo/edk2.git -b REVISION
 ```
 
-Clone official Dasharo EDK2 repository to your docker instance,
-with git or downloading the source code from github.
+Clone Dasharo EDK II Platforms fork:
 
 ```bash
-git clone https://github.com/Dasharo/edk2.git
+git clone https://github.com/Dasharo/edk2-platforms.git -b REVISION
```

Comment:
We do not develop edk2-platforms at all basically, so we don't plan to tag it

Reply:
If you wish to, you may tag this commit https://github.com/Dasharo/edk2-platforms/commit/3323ed481d35096fb6a7eae7b49f35eff00f86cf
with some semver for example.

Reply:
@miczyg1 the point is we have to tag because otherwise, we don't have a stable reference point, and we should have stable reference points for all firmware releases and all components. This is required for SBOMs and validation purposes.

> If you wish to, you may tag this commit https://github.com/Dasharo/edk2-platforms/commit/3323ed481d35096fb6a7eae7b49f35eff00f86cf
with some semver for example.

If you look at the commit, you can see it is tagged with `qemu_q35_v0.1.0-rc1`. I would like to move with what we have now because we are waiting to finalize #677 may block release progress for some time.

Reply:
I would drop the platform in the tag name, because edk2-platforms usage in Dasharo is not related to given platform.

Reply:
@miczyg1 it is not related right now. Creating multiple tag schemes adds a maintenance burden and unnecessarily complicates procedures for end users. What should be a tagging scheme for edk2-platforms?

My goal is to achieve consistency in documentation from developers' and users' perspectives. If you think this is not a valid claim and we should apply different tagging schemes or not tag some repositories at all, we should fix that as part of #677 and treat the solution proposed here as temporary.

Reply:
> What should be a tagging scheme for edk2-platforms?

Like I said, we do not actively develop edk2-platforms, so putting a tag like v1.0 and leaving it alone is sufficient

Reply:
IMHO, this unnecessarily breaks the scheme. But I fixed here is the fix: c2c9b94e750b

P.S. I don't like v1.0; that's why I added v0.9.0